### PR TITLE
Dig items for 13 zones

### PR DESCRIPTION
**Pull request by RunAwayDSP**
dig diglet dig!

### CODE REVIEW

**Comment by RunAwayDSP** [comment]
I hope this was the right branch.. these zones are now done
http://wiki.ffxiclopedia.org/wiki/Chocobo_Digging/Zones_A-M

**Comment by xdemolish** [comment]
diglet_dig,not master

File: scripts/zones/Carpenters_Landing/Zone.lua [hunk 1]
```diff
@@ -11,6 +11,61 @@ require("scripts/globals/zone");
...
+                    { 4447, 38, DIGREQ_NONE },
+                    { 695, 45, DIGREQ_NONE },
+                    { 4100, 59, DIGREQ_BURROW },
+                    { 4101, 59, DIGREQ_BURROW },
+                    { 690, 15, DIGREQ_BORE },
+                    { 1446, 8, DIGREQ_BORE },
+                    { 700, 23, DIGREQ_BORE },
+                    { 701, 8, DIGREQ_BORE },
+                    { 696, 30, DIGREQ_BORE },
+					{ 4570, 10, DIGREQ_MODIFIER },
```
Comment (xdemolish) [inline scripts/zones/Carpenters_Landing/Zone.lua]: dat indentation

File: scripts/zones/Jugner_Forest/Zone.lua [hunk 2]
```diff
@@ -12,6 +12,61 @@ require("scripts/globals/icanheararainbow");
...
+                    { 4447, 38, DIGREQ_NONE },
+                    { 695, 45, DIGREQ_NONE },
+                    { 4100, 59, DIGREQ_BURROW },
+                    { 4101, 59, DIGREQ_BURROW },
+                    { 690, 15, DIGREQ_BORE },
+                    { 1446, 8, DIGREQ_BORE },
+                    { 702, 23, DIGREQ_BORE },
+                    { 701, 8, DIGREQ_BORE },
+                    { 696, 30, DIGREQ_BORE },
+					{ 4570, 10, DIGREQ_MODIFIER },
```
Comment (xdemolish) [inline scripts/zones/Jugner_Forest/Zone.lua]: and that

File: scripts/zones/Konschtat_Highlands/Zone.lua [hunk 3]
```diff
@@ -11,6 +11,62 @@ require("scripts/globals/zone");
...
+                    { 868, 45, DIGREQ_NONE },
+                    { 642, 71, DIGREQ_NONE },
+                    { 4099, 147, DIGREQ_BURROW },
+                    { 4100, 147, DIGREQ_BURROW },
+                    { 845, 28, DIGREQ_BORE },
+                    { 842, 27, DIGREQ_BORE },
+                    { 843, 23, DIGREQ_BORE },
+                    { 1845, 22, DIGREQ_BORE },
+                    { 838, 19, DIGREQ_BORE },
+					{ 4570, 10, DIGREQ_MODIFIER },
```
Comment (xdemolish) [inline scripts/zones/Konschtat_Highlands/Zone.lua]: ok just change the tabs to 4 spaces on w/e looks weird in the diff

**Comment by Ex0r** [comment]
RunAway, you forgot to include the requires and load the packages at the top of the Zone.lua files, like this:

package.loaded["scripts/globals/chocobo_digging"] = nil;
require("scripts/globals/chocobo_digging");

**Comment by xdemolish** [comment]
don't need to reload it /every/ time, left that in there for testing

**Pull request closed without merging**